### ISSUE TITLE
As a project coordinator, I want to select a funding stream and RFP as the first step of creating a project

### ISSUE THREAD

**Issue by matthieu-foucault** [bcgov/cas-cif#1075]
#### Description:

[- Envisioned workflow for EP](https://www.figma.com/proto/PzOY8RyQnVXnjZJdSv2AqQ/CIF-Design?page-id=5936%3A294&node-id=5936%3A1172&viewport=-589%2C1434%2C0.25&scaling=scale-down&starting-point-node-id=5936%3A1172&show-proto-sidebar=1) (validated by CIF)
[- Envisioned workflow for IA](https://www.figma.com/proto/PzOY8RyQnVXnjZJdSv2AqQ/CIF-Design?page-id=5936%3A294&node-id=6008%3A2837&viewport=-589%2C1434%2C0.25&scaling=scale-down&starting-point-node-id=6008%3A3069&show-proto-sidebar=1) (validated by CIF)

This would prevent altering it later and having form records that are not relevant. 

#### Acceptance Criteria:

Given I am a CIF user
When I click 'create project'
Then I am prompted to enter the `stream` and `RFP year` before continuing to the rest of the forms

#### Development Checklist:

- [ ] The 'create project' button creates a project on click. This is already how it works, but it will still need to create the project before we can populate the values.
- [ ] User is prompted to enter `stream` and `RFP year` before continuing to the rest of the forms
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [x] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [x] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by LindsayMacfarlane** [comment]
@pbastia - does this need to be done before #1100? Or are they completely independent?

**Comment by pbastia** [comment]
@LindsayMacfarlane yes, this should be done before #1100

**Comment by LindsayMacfarlane** [comment]
Thank you @BCerki - looks great!

A couple of observations:
- I'm assuming there is a ticket to make the funding stream and RFP year immutable on the project overview page?
- We will need to add 2023 to the RFP year (do we need a ticket for that?)

**Comment by BCerki** [comment]
> Thank you @BCerki - looks great!
> 
> A couple of observations:
> 
> * I'm assuming there is a ticket to make the funding stream and RFP year immutable on the project overview page?
Yes, it's this one: https://github.com/bcgov/cas-cif/issues/1100
> * We will need to add 2023 to the RFP year (do we need a ticket for that?)
I've started a ticket for that here: https://github.com/bcgov/cas-cif/issues/1318
